test/unit/Breadcrumbs.test.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-env mocha */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import Breadcrumbs from '../../src/app/components/Breadcrumbs/Breadcrumbs';
+
+describe('Breadcrumbs', () => {
+  describe('On render all crumbs', () => {
+    let component;
+    const links = [{
+      href: '/search?q=journey&field=keyword',
+      text: 'Search Results',
+    },
+    {
+      href: '/work?workId=foobar',
+      text: 'Work Details',
+    }];
+    const pageType = 'details';
+
+    before(() => {
+      component = shallow(<Breadcrumbs links={links} pageType={pageType} />);
+    });
+
+    it('should render a nav and ol element', () => {
+      expect(component.find('nav').length).to.equal(1);
+      expect(component.find('ol').length).to.equal(1);
+    });
+
+    it('should have three list items and two links', () => {
+      expect(component.find('li').length).to.equal(3);
```

Comment:
Maybe being more explicit about what values these are will be better so you know that you have three li items but only two `Link`s. You have tests below for the `href` prop but having it will be nice too.